### PR TITLE
refact(docs): update Project README and contributors docs

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ If you want to build the cstor-operators right away then the following is the st
...
-cd cstor-operators
-make all.amd64
+mkdir -p $GOPATH/src/github.com
+cd $GOPATH/src/github.com
+git clone https://github.com/openebs/cstor-operators openebs/cstor-operators
+cd openebs/cstor-operators
+make all
 ```
 
 Alternatively, you can open this repo in Gitpod and mkae your PR right from the browser:
```
Comment (sonasingh46) [inline CONTRIBUTING.md:19]: typo : 
`mkae` ---------> `make`

File: README.md [hunk 2]
```diff
@@ -1,17 +1,15 @@
-# cstor-operators
+# CStor-Operators
```
Comment (kmova) [inline README.md:1]:
```suggestion
# cStor-Operators
```

File: README.md [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
 
 <img width="300" align="right" alt="OpenEBS Logo" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/stacked/color/openebs-stacked-color.png" xmlns="http://www.w3.org/1999/html">
 
-Collection of enhanced Kubernetes Operators for managing OpenEBS cStor Data Engine.
-
 ## Project Status: Beta
 
-The cStor operators work in conjunction with the [cStor CSI driver](https://github.com/openebs/cstor-csi) to
-provide cStor volumes for stateful workloads.
+We are always happy to list users who run CStor in production, checkout our existing [adopters](https://github.com/openebs/openebs/tree/master/adopters) and their [feedbacks](https://github.com/openebs/openebs/issues/2719).
```
Comment (kmova) [inline README.md:12]:
```suggestion
We are always happy to list users who run cStor in production, check out our existing [adopters](https://github.com/openebs/openebs/tree/master/adopters), and their [feedbacks](https://github.com/openebs/openebs/issues/2719).
```

**Review by mittachaitu — APPROVED** [review]
LGTM

**Review by kmova — APPROVED** [review]

**Pull request merged**